File: app/src/main/java/com/panwar2001/pdf2txt/onBoard.java
package com.panwar2001.pdf2txt;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.google.android.gms.ads.AdRequest;
import com.panwar2001.pdf2txt.databinding.ActivityOnBoardBinding;

public class onBoard extends AppCompatActivity {
    ActivityOnBoardBinding binding;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityOnBoardBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.onBoardButton.setOnClickListener(view->{
            Intent intent=new Intent(onBoard.this,MainActivity.class);
            startActivity(intent);
        });
        // admob code
        AdRequest adRequest = new AdRequest.Builder().build();
        binding.adView.loadAd(adRequest);
    }
}

File: app/src/main/java/com/panwar2001/pdf2txt/OnBoard.kt
package com.panwar2001.pdf2txt

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.google.android.gms.ads.AdRequest
import com.panwar2001.pdf2txt.databinding.ActivityOnBoardBinding

class OnBoard : AppCompatActivity() {
    private var binding: ActivityOnBoardBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityOnBoardBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
        binding!!.onBoardButton.setOnClickListener { _: View? ->
            val intent = Intent(this@OnBoard, MainActivity::class.java)
            startActivity(intent)
        }
        // admob code
        val adRequest = AdRequest.Builder().build()
        binding!!.adView.loadAd(adRequest)
    }
}